srtcore/core.cpp

```diff
@@ -559,37 +559,37 @@ void CUDT::setOpt(SRT_SOCKOPT optName, const void* optval, int optlen)
         break;
 
     case SRTO_TSBPDMODE:
-        if (m_bOpened)
-            throw CUDTException(MJ_NOTSUP, MN_ISBOUND, 0);
+        if (m_bConnected)
+            throw CUDTException(MJ_NOTSUP, MN_ISCONNECTED, 0);
 
         m_bOPT_TsbPd = cast_optval<bool>(optval, optlen);
         break;
 
     case SRTO_LATENCY:
-        if (m_bOpened)
-            throw CUDTException(MJ_NOTSUP, MN_ISBOUND, 0);
+        if (m_bConnected)
+            throw CUDTException(MJ_NOTSUP, MN_ISCONNECTED, 0);
 
         m_iOPT_TsbPdDelay     = cast_optval<int>(optval, optlen);
```

Comment:
I guess if a connection is in progress, the latency should also not be allowed to change.

Reply:
In the PR where I'm preparing the separate configuration object I'm preparing also an array with option flags where the PRE option is defined by the flag - and PRE means that both m_bConnecting and m_bConnected must be false. Actually I don't think the space between connecting and connected should be allowed for any option that is state-restricted.

Reply:
`(m_bConnected || m_bConnecting)` then?

Reply:
Yes, all those options that are restricted as PRE will be allowed only under this condition.

Reply:
I am confused.
In this PR you change ` if (m_bOpened)` to `if (m_bConnected)`.
Why not to `if (m_bConnected || m_bConnecting)`?